File: EventTime/src/main/java/com/hakankuru/EventTime/dto/EventResponse.java
package com.hakankuru.EventTime.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class EventResponse {
    private Long eventId;
    private String title;
    private String description;
    private String image;
    private LocalDateTime startAt;
    private LocalDateTime endAt;
    private LocalDateTime applyStartAt;
    private LocalDateTime applyEndAt;
    private Integer capacity;
    private Long clubId;
    private String clubName;
}

File: eventhub/app/src/main/java/com/hakankuru/eventhub/data/remote/response/EventResponse.kt
package com.hakankuru.eventhub.data.remote.response

data class EventResponse(
    val eventId: Long,
    val title: String,
    val description: String,
    val image: String?,
    val startAt: String,
    val endAt: String?,
    val applyStartAt: String?,
    val applyEndAt: String?,
    val capacity: Int?,
    val clubId: Long,
    val clubName: String
)
